src/main/java/org/openstreetmap/atlas/tags/MinHeightTag.java

```diff
@@ -0,0 +1,38 @@
+package org.openstreetmap.atlas.tags;
+
+import java.util.Optional;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.Tag.Validation;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+import org.openstreetmap.atlas.tags.annotations.validation.DoubleValidator;
+import org.openstreetmap.atlas.utilities.scalars.Height;
+
+/**
+ * OSM min_height tag
+ *
+ * @author ajayaswal
+ */
+@Tag(value = Validation.LENGTH, taginfo = "https://taginfo.openstreetmap.org/keys/min_height#values", osm = "https://wiki.openstreetmap.org/wiki/Key:min_height")
+public interface MinHeightTag
+{
+    @TagKey
+    String KEY = "min_height";
+
+    DoubleValidator validator = new DoubleValidator();
+
+    static Optional<Height> get(final Taggable taggable)
+    {
+        final Optional<String> tagValue = taggable.getTag(KEY);
+
+        if (tagValue.isPresent())
+        {
+            if (validator.isValid(tagValue.get()))
+            {
+                return Optional.of(new Height(Double.valueOf(tagValue.get())));
+            }
```

Comment:
Nit: possible improvement could be combining if statements:
`if (tagValue.isPresent() && validator.isValid(tagValue.get())) {...}`